src/lib/spinel/spinel.h

```diff
@@ -1517,6 +1517,7 @@ enum
     SPINEL_PROP_PHY_PCAP_ENABLED   = SPINEL_PROP_PHY__BEGIN + 8,  ///< [b]
     SPINEL_PROP_PHY_CHAN_PREFERRED = SPINEL_PROP_PHY__BEGIN + 9,  ///< [A(C)]
     SPINEL_PROP_PHY_FEM_LNA_GAIN   = SPINEL_PROP_PHY__BEGIN + 10, ///< dBm [c]
+    SPINEL_PROP_PHY_CHAN_MAX_POWER = SPINEL_PROP_PHY__BEGIN + 11, ///< [cC]
```

Comment:
Suggest adding more documentation/detail for this.

Mainly adding  that this is a "write only" property and describing the format `[cC]` and that it  represents "channe"l and then "max power".

BTW, I think it should be `"[Cc]"` instead. `C` is `UINT8` for channel.